processes/README.md

```diff
@@ -0,0 +1,7 @@
+# Guides
```

Comment:
```suggestion
# Processes
```

Reply:
Alan

I'm not a  Github expert, but I've just returned from holiday and this morning started to look at the repo and found it all very confusing. So anything you can do to make it easier to navigate would be very appreciated. 

That said please remember this is supposed to be a temporary repo ahead of a project to look at the longer term solution so please dont spend too long on this